docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
```

Comment:
does it need a db at this point?

Reply:
Also, from what I remember of the conversation yesterday, provisioning a DB on the infrastructure requires a separate PR with cloud platform team. It might be easier to wait until we need it.

Reply:
i think so rails will fall over at init time as it makes connections to the db right away. I wanted to commit a docker image that runs

Reply:
the only way to remove the db is to tare out active record from rails